code/game/gamemodes/wizard/magic_tarot.dm

```diff
@@ -576,6 +605,7 @@
 /datum/tarot/the_sun
 	name = "XIX - The Sun"
 	desc = "May the light heal and enlighten you."
+	extended_desc = "heals you back to the users peak strength. Some say this card holds the power of a god."
```

Comment:
```suggestion
	extended_desc = "heals the user back to the their peak strength. Some say this card holds the power of a god."
```
Not sure that last bit fits super well, but I'm trying to just do grammar and sentence flow corrections here.